Scenarios/TripPin/src/webapi/ODataSamples.WebApiService/App_Start/WebApiConfig.cs

```diff
@@ -29,6 +29,10 @@ private static IEdmModel GetEdmModel()
             builder.EntitySet<Airline>("Airlines");
             builder.Singleton<Person>("Me");
 
+            // Customer - Order scenarios
+            builder.EntitySet<Customer>("Customers");
```

Comment:
Does TripPin have customers and orders?  We shall have a coherent api sample service.